scipy/linalg/tests/test_matfuncs.py

```diff
@@ -838,3 +838,23 @@ def test_expm_cond_fuzz(self):
             # eps times the condition number kappa.
             # In the limit as eps approaches zero it should never be greater.
             assert_array_less(p_best_relerr, (1 + 2*eps) * eps * kappa)
+
+
+class TestKhatriRao(object):
+
+    def test_basic(self):
+        a = khatri_rao(array([[1, 2], [3, 4]]),
```

Comment:
Use distinct arrays to make the debugging easier.

Reply:
ok